.gitignore

```diff
@@ -1,3 +1,4 @@
 hydra-booster
 hydra-belly
 coverage.txt
+data
```

Comment:
can prometheus write to prometheus-data?

Reply:
I did not see an option for this I could put in `promconfig.yaml` but there is a command line option I could add in the docs (I wanted this also 😂).